dpytools/http/upload/token_auth.py

```diff
@@ -107,6 +116,14 @@ def refresh_user_token(self):
         if response.status_code == 201:
             self.auth_token = response.headers["Authorization"]
             self.id_token = response.headers["ID"]
+
+            logger.info(
+                "User tokens refreshed",
+                data={
+                    "token_refresh_url": token_refresh_url,
+                    "response_headers": response.headers,
+                },
+            )
         else:
             err = Exception(
```

Comment:
Could there be a test for when the response fails (not code 201) when refreshing user token? To make sure the error appears when expected.